lms/djangoapps/student_dashboard/test/view_tests.py

```diff
@@ -0,0 +1,47 @@
+import ddt
```

Comment:
Please add module docstring. 

Please run pylint by using the following command and fix your code related issues

`paver run_pylint --system==lms/djangoapps/student_dashboard`